Doc/library/collections.rst

```diff
@@ -1175,6 +1176,11 @@ attribute.
     subclass) or an arbitrary sequence which can be converted into a string using
```

Comment:
This phrase is outdated (remained from Python 2). Virtually any object (not just sequences) can be converted into a string using str(). However bytes is bad example, it emits a warning or an exception when converted to str.